frontend/src/components/TodoCard/index.js

```diff
@@ -119,9 +118,20 @@ class TodoCard extends Component {
     this.setCardStatus('editable');
   }
 
+  handleHover(isHover) {
+    const nextState = isHover ? 'removable' : 'idle';
+    return (e) => {
+      const target = e.target;
+      if (target && this.state.cardStatus !== nextState) this.setCardStatus(nextState);
+    };
+  }
+
```

Comment:
특별히 고차함수를 쓰신 이유가 있을까요?? 😀

Reply:
음 저 부분을 고차함수를 사용하지 않는다면 이런 모습일 것 같아요!
```suggestion
  handleHover(e, isHover) {
    const target = e.target;
    const nextState = isHover ? 'removable' : 'idle';
    if (target && this.state.cardStatus !== nextState) this.setCardStatus(nextState);
  }
```

평소에 파라미터에 따라서 다른 핸들링을 해야하는 경우에 고차함수를 사용하고는 하는데 
이 부분에서는 고차함수를 써서 얻는 장점이 크지는 않은 것 같네요!
파라미터에 두개를 명시하지 않는다 정도..일 것 같습니다 :)